### PR TITLE
[Doc] Add tutorials/customize_models.md Chinese version

### PR DESCRIPTION
**Pull request by wHao-Wu**
* Add tutorials/customize_models.md Chinese version

### CODE REVIEW

File: docs_zh-CN/tutorials/customize_models.md [hunk 1]
```diff
@@ -1 +1,512 @@
 # 教程 4: 自定义模型
+
+我们通常把模型的各个组成成分分成6种类型：
+
+- encoder：包括 voxel layer、voxel encoder 和 middle encoder 等进入 backbone 前所使用的基于 voxel 的方法，如 HardVFE 和 PointPillarsScatter。
+- backbone：通常采用 FCN 网络来提取特征图，如 ResNet 和 SECOND。
+- neck：位于 backbones 和 heads 之间的组成模块，如 FPN 和 SECONDFPN。
+- head：用于特定任务的组成模块，如检测框的预测和掩码的预测。
+- roi extractor：用于从特征图中提取 RoI 特征的组成模块，如 H3DRoIHead 和 PartAggregationROIHead。
+- loss：heads 中用于计算损失函数的组成模块，如 FocalLoss、L1Loss 和 GHMLoss。
```
Comment (Tai-Wang) [inline docs_zh-CN/tutorials/customize_models.md:10]: 可以翻译一下各个组成成分，比如 编码器 (encoder)

**Review by Tai-Wang — COMMENTED** [review]
Almost ready to merge

File: docs_zh-CN/tutorials/customize_models.md [hunk 2]
```diff
@@ -1 +1,512 @@
...
+    voxel_encoder=dict(
+        type='HardVFE',
+        arg1=xxx,
+        arg2=xxx),
+    ...
+```
+
+### 添加新建 backbone
+
+接下来我们以 SECOND（Sparsely Embedded Convolutional Detection） 为例展示如何开发新的组成模块。
```
Comment (Wuziyi616) [inline docs_zh-CN/tutorials/customize_models.md:68]: 给 SECOND 加个 citation？因为[这里](https://github.com/open-mmlab/mmdetection3d/pull/820/files#diff-7bab21b4db14f8f80eb0bb995afe67c62dbede90efc711365d8134f5c6ffb556R177) PartA2 加了

Comment (Wuziyi616) [reply docs_zh-CN/tutorials/customize_models.md:68]: Please also modify the corresponding English doc

File: docs_zh-CN/tutorials/customize_models.md [hunk 3]
```diff
@@ -1 +1,512 @@
...
+- roi 提取器（roi extractor）：用于从特征图中提取 RoI 特征的组成模块，如 H3DRoIHead 和 PartAggregationROIHead。
+- 损失函数（loss）：heads 中用于计算损失函数的组成模块，如 FocalLoss、L1Loss 和 GHMLoss。
+
+## 开发新的组成模块
+
+### 添加新建 encoder
+
+接下来我们以 HardVFE 为例展示如何开发新的组成模块。
+
+#### 1. 定义一个新的 voxel encoder（如 HardVFE）
```
Comment (Wuziyi616) [inline docs_zh-CN/tutorials/customize_models.md:18]: HardVFE 加一个注释？

**Review by Wuziyi616 — COMMENTED** [review]
Good job

File: docs_zh-CN/tutorials/customize_models.md [hunk 4]
```diff
@@ -1 +1,512 @@
 # 教程 4: 自定义模型
+
+我们通常把模型的各个组成成分分成6种类型：
+
+- 编码器（encoder）：包括 voxel layer、voxel encoder 和 middle encoder 等进入 backbone 前所使用的基于 voxel 的方法，如 HardVFE 和 PointPillarsScatter。
+- 骨干网络（backbone）：通常采用 FCN 网络来提取特征图，如 ResNet 和 SECOND。
+- 颈部网络（neck）：位于 backbones 和 heads 之间的组成模块，如 FPN 和 SECONDFPN。
+- 检测头（head）：用于特定任务的组成模块，如检测框的预测和掩码的预测。
+- roi 提取器（roi extractor）：用于从特征图中提取 RoI 特征的组成模块，如 H3DRoIHead 和 PartAggregationROIHead。
```
Comment (ZwwWayne) [inline docs_zh-CN/tutorials/customize_models.md:9]: RoI

**Review by ZwwWayne — APPROVED** [review]

**Pull request merged**